python/source/conf.py

```diff
@@ -0,0 +1,57 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Apache Arrow Python Cookbook'
+copyright = '2021, Apache Software Foundation'
+author = 'Apache Software Foundation'
```

Comment:
nit: **The** Apache Software Foundation

Reply:
Should it be addressed in arrow docs too? I remember copying it from there ( https://github.com/apache/arrow/blob/master/docs/source/conf.py#L117 )